### ISSUE TITLE
Link to documentation in readme is broken

### ISSUE THREAD

**Issue by meidlinga** [laminas/laminas-cache-storage-adapter-redis#28]
#### Summary

Link to documentation https://docs.laminas.dev/laminas-cache-storage-adapter-redis/ in readme is broken and leads to a 404 on github pages.

**Comment by boesing** [comment]
Yes, this stands for all adapters. Sorry for this. 
You can find documentation at https://docs.laminas.dev/laminas-cache/v3/storage/adapter/

We are still investigating a proper way to refactor the documentation as it is currently not that optimized.

**Comment by meidlinga** [comment]
Thanks for the link. Do you mean the laminas-cache alternative in general? I think linking https://docs.laminas.dev/laminas-cache/v3/storage/adapter/#redis-adapter would be a good alternative.

**Comment by boesing** [comment]
@meidlinga Yah, but we would prefer to have dedicated pages for every cache adapter.
But since we have ripped out the cache adapters last year, we haven't had time to investigate further documentation changes.

Maybe at some point we will provide redirects for the supported cache adapters to their new pages within the documentation website.

As of now, all documentations regarding any cache adapter is part of `laminas-cache` itself (in its repository).
The idea is to move all adapter specific documentations to the adapter repositories and then merge them back to the cache adapter when building docs. Sadly, as of now, it always requires a release for `laminas-cache` to actually build documentation. Thats one of the things we are still investigating as I don't see that we create patch releases for `laminas-cache` just to roll out documentation releases of new versions of several cache adapters.

I keep this ticket open to keep track of the fact that the link to the documentation is broken. 
If you want to, you can send a patch to fix the README file. I'm open to merge that to the latest minor branch 👍🏼

**Comment by froschdesign** [comment]
All these links are wrong in the readme files:

* https://docs.laminas.dev/laminas-cache-storage-adapter-test/
* https://docs.laminas.dev/laminas-cache-storage-adapter-apc/
* https://docs.laminas.dev/laminas-cache-storage-adapter-apcu/
* https://docs.laminas.dev/laminas-cache-storage-adapter-blackhole/
* https://docs.laminas.dev/laminas-cache-storage-adapter-dba/
* https://docs.laminas.dev/laminas-cache-storage-adapter-ext-mongodb/
* https://docs.laminas.dev/laminas-cache-storage-adapter-filesystem/
* https://docs.laminas.dev/laminas-cache-storage-adapter-memcache/
* https://docs.laminas.dev/laminas-cache-storage-adapter-memcached/
* https://docs.laminas.dev/laminas-cache-storage-adapter-memory/
* https://docs.laminas.dev/laminas-cache-storage-adapter-mongodb/
* https://docs.laminas.dev/laminas-cache-storage-adapter-redis/
* https://docs.laminas.dev/laminas-cache-storage-adapter-session/
* https://docs.laminas.dev/laminas-cache-storage-adapter-wincache/
* https://docs.laminas.dev/laminas-cache-storage-adapter-xcache/
* https://docs.laminas.dev/laminas-cache-storage-adapter-zend-server/
* https://docs.laminas.dev/laminas-servicemanager-migration/

@boesing 
> The idea is to move all adapter specific documentations to the adapter repositories and then merge them back to the cache adapter when building docs.

I am still working on the problem and am currently testing something like this:

```yaml
- Redis: '!import https://github.com/laminas/laminas-cache-storage-adapter-redis?branch=…?docs_dir=…'
```

**Comment by froschdesign** [comment]
Related to https://github.com/laminas/laminas-continuous-integration-action/issues/34

**Comment by meidlinga** [comment]
@boesing , @froschdesign : I opened a PR for a temporary fix, as you suggested. Thanks for the detailed explanation of the underlying problem.

**Comment by boesing** [comment]
Closed with #29
